storm-data-contracts-testng/src/main/java/com/forter/contracts/testng/TestDataProvider.java

```
@@ -108,7 +110,12 @@ else if (textNode.textValue().equals("__NULL__")) {
     }
 
     private static Object createAndValidateContract(ObjectNode node, ContractFactory<?> factory) {
-        Object contract = ContractConverter.instance().convertObjectNodeToContract(node, factory);
+        Object contract;
+        try {
+            contract = ContractConverter.instance().convertObjectNodeToContract(node, factory);
+        } catch (JsonProcessingException e) {
+            contract = null;
```

why not rethrow the exception ?